blob/src/test/java/io/crate/integrationtests/RecoveryTests.java

```diff
@@ -59,7 +60,7 @@
 import static org.hamcrest.core.IsEqual.equalTo;
 
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.SUITE, numDataNodes = 0, numClientNodes = 0, transportClientRatio = 0)
-@ThreadLeakFilters(filters = {RecoveryTests.RecoveryTestThreadFilter.class})
+@ThreadLeakFilters(filters = {RecoveryTests.RecoveryTestThreadFilter.class, NettyLeakDetectionFilter.class})
```

Comment:
isn't this already annotated on the parents class?

Reply:
The annotation from the parent class only has an effect if the sub-classes don't have their own annotation. Here we already have a `RecoveryTestThreadFilter` in addition, so I had to extend it here.

Reply:
Thanks for the clarification.